code/modules/mob/living/simple_animal/animals/giant_spider.dm

```diff
@@ -78,18 +78,6 @@ Nurse Family
 	var/atom/cocoon_target
 	var/egg_inject_chance = 5
 
-/mob/living/simple_animal/hostile/giant_spider/nurse/medical
-	desc = "Furry and beige, it makes you shudder to look at it. This one has brilliant green eyes and a tiny nurse hat."
```

Comment:
these dudes being less dangerous was intentional and i'd love you forever if we kept at least one; they're a nice unique feature that you get to see at least on occasion, but i feel having even two is overuse

Reply:
Leave 'em in the code. Good for adminbuse!